src/ConductorSharp.Engine/Builders/WorkflowDefinitionBuilder.cs

```diff
@@ -111,7 +122,7 @@ public WorkflowDefinition Build()
 
             return new WorkflowDefinition
             {
-                Name = _name,
+                Name = BuildContext.WorkflowName,
```

Comment:
The field `_name` is no longer used since we take name form BuildContext, and should be removed.

Reply:
Done